go/flags/endtoend/vtctld.txt

```diff
@@ -3,8 +3,9 @@ Usage of vtctld:
       --alsologtostderr                                                  log to standard error as well as files
       --azblob_backup_account_key_file string                            Path to a file containing the Azure Storage account key; if this flag is unset, the environment variable VT_AZBLOB_ACCOUNT_KEY will be used as the key itself (NOT a file path).
       --azblob_backup_account_name string                                Azure Storage Account name for backups; if this flag is unset, the environment variable VT_AZBLOB_ACCOUNT_NAME will be used.
+      --azblob_backup_buffer_size int                                    The memory buffer size to use in bytes, per file or stripe, when streaming to Azure Blob Service. (default 104857600)
```

Comment:
I wonder if these two flags are actually needed for vtctld. I would guess not, because vtctld only needs to access the storage backend for listing / deleting backups.
But it will be a hassle to break these out because all flags in the packages are being registered as a group.